incubator/dcos_spark_install/preparation/script.md

```diff
@@ -0,0 +1,55 @@
+# Creating a Cluster
+
+We will use the Azure CLI 2.0 to quickly create an Azure Container
+Services cluster. Ensure you have the Azure CLI installed and have
+logged in using the `az login` command.
+
+We first need to set a few environment variables we will use later:
+
+```
+RESOURCE_GROUP=acs-spark-demo
+SERVICE_NAME=acs-dcos-spark
+DNS_NAME_PREFIX=acs-dcos-spark
+REGION=eastus
+```
+
+```
+echo region is $REGION
+```
```

Comment:
I've opened up a PR that adds support for persistent environment variables via a `env.json` file (https://github.com/rgardler/simdem/pull/1)